Update app-service-mobile-how-to-configure-active-directory-authentic…

mikeehsu opened the pull request:
…ation.md

Updated the documentation to reflect changes to the Azure Active Directory in the portal (moved from classic).

I created new screenshots to go along with the documentation changes. I was not able to upload them, so I have attached them to my pull request. If there is a different/better way to do this let me know.

Thanks,

Mike


![app-service-webapp-auth-config](https://user-images.githubusercontent.com/11183172/30832312-a8822d72-a218-11e7-8a77-300e884dd4fc.png)
![app-service-webapp-url](https://user-images.githubusercontent.com/11183172/30832313-a884acb4-a218-11e7-8647-8ce3725f1f79.png)
![app-service-aad-app_registration](https://user-images.githubusercontent.com/11183172/30832309-a86e112a-a218-11e7-8dc5-fd55e4f724c2.png)
![app-service-aad-app-registration-config-appidurl](https://user-images.githubusercontent.com/11183172/30832311-a873f568-a218-11e7-8388-0507d72066a6.png)
![app-service-aad-app-registration-config-replyurl](https://user-images.githubusercontent.com/11183172/30832310-a8708a18-a218-11e7-8df1-6ba576a1ec7f.png)
![app-service-aad-app-registration-create](https://user-images.githubusercontent.com/11183172/30832314-a886e5c4-a218-11e7-965c-3aa4a0be7502.png)
![app-service-aad-endpoints](https://user-images.githubusercontent.com/11183172/30832315-a8934e90-a218-11e7-83c5-629e8d23c702.png)
![app-service-aad-endpoints-fedmetadataxml](https://user-images.githubusercontent.com/11183172/30832316-a893db76-a218-11e7-827d-de1d129f5756.png)
![app-service-webapp-auth](https://user-images.githubusercontent.com/11183172/30832317-a8965248-a218-11e7-84c0-b07c6791ee1d.png)

azurecla commented:
Hi __@ClarityQueen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-diree). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

Blackmist commented:
Looks good. Merge.

mikeehsu committed:
Update app-service-mobile-how-to-configure-active-directory-authentication.md

Updated the documentation to reflect changes to the Azure Active Directory in the portal (moved from classic).
